### PR TITLE
gauge 0.8.0.1

### PR DESCRIPTION
**Pull request by kashishm**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/gauge.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@
 class Gauge < Formula
   desc "Test automation tool that supports executable documentation"
   homepage "http://getgauge.io"
-  url "https://github.com/getgauge/gauge/archive/v0.7.0.tar.gz"
-  sha256 "959ad3feea738d107e8840804b8faf82d1c1242db8beb1f746e21c76d7277f09"
+  url "https://github.com/getgauge/gauge/archive/v0.8.0.1.tar.gz"
```
Comment (JCount) [inline Formula/gauge.rb]: Please update the version numbers in your PR (commit message and PR title) to accurately reflect the exact version being built.

**Review by JCount — CHANGES_REQUESTED** [review]
Please also update the `gauge_screenshot` `go_resource` to the most recent revision. Thanks.

File: Formula/gauge.rb [hunk 2]
```diff
@@ -43,6 +43,6 @@ def install
   end
 
   test do
-    assert_match version.to_s, shell_output("#{bin}/gauge -v")
+    assert_match version.to_s[0, 5], shell_output("#{bin}/gauge -v")
```
Comment (ilovezfs) [inline Formula/gauge.rb]: this is a bit unusual. what was the problem?

Comment (kashishm) [inline Formula/gauge.rb]: `gauge -v` shows version in the format `MAJOR.MINOR.PATCH` which is `0.8.0` but the version in this tag is `0.8.0.1`. So, I am using this to match `0.8.0.1` with `0.8.0`(`MAJOR.MINOR.PATCH`).

**Review by JCount — APPROVED** [review]

**Comment by JCount** [comment]
Thank you for your contribution, @kashishm!

**Pull request closed without merging**